packages/core-interop/api/presentation-core-interop.exports.csv

```diff
@@ -1,7 +1,6 @@
 sep=;
 Release Tag;API Item
 beta;createECSqlQueryExecutor(imodel: ICoreECSqlReaderFactory): IECSqlQueryExecutor
-beta;createHierarchyLevelDescriptor
```

Comment:
any idea why this disappeared?

Reply:
Not sure. I thought it might have something to do with mixed dependencies in `node_modules` after switching branches but looks like CI pipeline is ok with this change.

Reply:
Adding return type instead of using plain object fixed this.